docs/deployment.md

```diff
@@ -2,22 +2,12 @@
 
 ## Before you merge
 
-This app doesn't have CI setup, and it would be misleading to do so: the behaviour of the tests changes depending on the environment they are run in. You should manually test in applicable environments, until you are confident your change is not a breaking one.
-
-Example steps to test in Integration:
-
-- Click "Configure" for the [Smokey job][]
-- Change the "Branch specifier" to the name of your branch
-- Run a new build of the Smokey project and check the results
```

Comment:
I'd leave this as-is until https://trello.com/c/FPbuquwu/2953-investigate-setting-up-ci-for-smokey-5 is completed.

There's an argument for saying a lot of the above is still valid.